README.md

```diff
@@ -179,3 +221,21 @@ sudo ./build.sh  # or ./build-docker.sh
 If you wish to build further configurations upon (for example) the lite
 system, you can also delete the contents of `./stage3` and `./stage4` and
 replace with your own contents in the same format.
+
+
+## Skipping stages to speed up development
+
+If you're working on a specific stage the recommended development process is as
+follows:
+
+ * Remove the EXPORT_IMAGE and EXPORT_NOOBS files until you're ready to actually
+   generate an image
+ * Add SKIP files to the stages you don't want to build. For example, if you're
+   basing your image on the lite image you would add these to stages 3, 4 and 5.
```

Comment:
I need to re-test this to confirm, but I'm pretty sure if you're trying to validate your application will build correctly in stage 4 (for example), and you don't need to rebuild earlier stages, you can add SKIP files to stage 1, 2, and 3.

Reply:
That's what it says a couple of lines down, but you need to build the earlier stages first to get the build directories in place